.travis.yml

```diff
@@ -1,11 +1,11 @@
+osx_image: xcode9
 xcode_sdk: iphonesimulator11.0
 language: objective-c
 cache: cocoapods
 podfile: Example/Podfile
 before_install:
-- gem install cocoapods # Since Travis is not always on latest version
```

Comment:
Added spaces here to conform to yaml syntax for sequences.  Yaml depends on spaces for parsing. This is a decent resource: https://learnxinyminutes.com/docs/yaml/

Reply:
@grocky why was -- gem install cocoapods not needed anymore?

Reply:
Travis already has cocoapods installed. I think the dependency differences you have have been encountering were from the xcode version differences.